web-wallet/src/lib/dusk/currency/__tests__/luxToDusk.spec.js

```diff
@@ -3,11 +3,6 @@ import { describe, expect, it } from "vitest";
 import { luxToDusk } from "..";
 
 describe("luxToDusk", () => {
-  it("should convert a number amount in Lux to Dusk", () => {
-    expect(luxToDusk(1e9)).toBe(1);
-    expect(luxToDusk(123_456_789_012)).toBe(123.456789012);
-  });
-
   it("should accept `BigInt`s as input", () => {
```

Comment:
Maybe rewrite this test description now, as BigInt is the only option and not an additional one anymore.